pagerduty/resource_pagerduty_escalation_policy.go

```diff
@@ -172,10 +172,23 @@ func fetchEscalationPolicy(d *schema.ResourceData, meta interface{}, errCallback
 
 	o := &pagerduty.GetEscalationPolicyOptions{Includes: []string{"escalation_rule_assignment_strategies"}}
 
+	var escalationPolicyFirstAttempt *pagerduty.EscalationPolicy
+
+	escalationPolicyFirstAttempt, _, err = client.EscalationPolicies.Get(d.Id(), o)
+	if err != nil && isErrCode(err, http.StatusForbidden) {
+		// Removing the inclusion of escalation_rule_assignment_strategies for
+		// accounts wihtout the required entitlements.
+		o = nil
+	}
+
+	if err == nil && escalationPolicyFirstAttempt == nil {
+		return setResourceEPProps(d, escalationPolicyFirstAttempt)
```

Comment:
This seems wrong, should be `if err == nil && escalationPolicyFirstAttempt != nil {` instead of `if err == nil && escalationPolicyFirstAttempt == nil {` due to `setResourceEPProps(d *schema.ResourceData, escalationPolicy *pagerduty.EscalationPolicy)` function is not checking if escalationPolicy is nil.